gigareview/console/models/ManuscriptsWorker.php

```diff
@@ -16,7 +18,33 @@ public function execute($queue)
     {
 
     }
-    
+
+    /**
+     * @param string $manuscriptPath
+     * @return array
+     */
+    public function parseManuscriptReport(string $manuscriptPath): array
+    {
+        $emReportJob = new EMReportJob();
+        $manuscriptData = $emReportJob->parseReport($manuscriptPath);
+        return ($manuscriptData);
+    }
```

Comment:
If an `EMReportJob` object is created, I would expect to see it pushed into a queue but it is not used in this way here.  For example, see how `EMReportJob` is used in `FetchReportsController` in `actionFetch()`. Instead, you could call `parseReport($content)` from within the `execute($queue)` function then you won't need this `parseManuscriptReport(string $manuscriptPath)` function.